### DISCUSSION TITLE
How to change cursor color?

### DISCUSSION THREAD

**Discussion by WernerWessely** [NvChad/NvChad#1190] [Q&A] [answered]
Hi everyone and thanks for this great project, finally an easy way to start a config with some sane defaults.

I have been struggling with one small point: How can I change the cursor color?

I have managed to set my theme and and enable the cursorline:

```lua
M.ui = {
	-- set the nord theme
	theme = "nord",
	-- enable the cursor line and change cursor color
	hl_override = {
		Cursor = { bg = "one_bg", fg = "one_bg" },
		CursorLine = { bg = "one_bg" },
	},
}
```
The Cursor member does not seem to have any effect, anybody know how to achieve this?

Thanks a lot!

**Comment by DougAnderson444** [comment]
Hey, did you ever figure this out? I can't seem to change the Cursor color either

**Comment by dev9nine** [comment, marked as answer]
usually it's the setting for the terminal.
and not every terminal support this.
here how i did it for kitty (in kitty.conf):
cursor_shape     	block
cursor		 	#bc260e
cursor_text_color 	#111111
